ui/page/send/send_confirm_modal.go

```diff
@@ -156,9 +156,9 @@ func (scm *sendConfirmModal) Layout(gtx layout.Context) D {
 					return layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
 						layout.Rigid(func(gtx C) D {
 							icon := scm.Icons.NavigationArrowForward
-							icon.Color = scm.Theme.Color.Gray3
 							return layout.Inset{Right: values.MarginPadding8}.Layout(gtx, func(gtx C) D {
-								return icon.Layout(gtx, values.MarginPadding15)
+								// todo:iconsize values.MarginPadding8
```

Comment:
if the todos for icon size are going to be done in a later PR then an issue should be opened so it's not forgotten